text/src/autogluon/text/automm/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -76,12 +78,14 @@ model:
     train_transform_types:
       - "resize_shorter_side"
       - "center_crop"
+      - "trivial_augment"
```

Comment:
Should we put `trivial_augment` at the beginning or the end of this list?

Reply:
One drawback of data augmentation after cropping is that it may introduce black margins when performing the spatial augmentations, e.g, rotation.

Reply:
But it is the common practice?

Reply:
One example is the implementation in TIMM: https://github.com/rwightman/pytorch-image-models/blob/ef72ad417709b5ba6404d85d3adafd830d507b2a/timm/data/transforms_factory.py#L44-L76